src/test/suite/fixtures/recorded/languages/php/changeFunk2.yml

```diff
@@ -0,0 +1,31 @@
+languageId: php
+command:
+  version: 1
+  spokenForm: change funk
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: namedFunction, includeSiblings: false}
+initialState:
+  documentContents: |-
+    <?php
+
+    $myFunc = function() {
+
+    };
+  selections:
+    - anchor: {line: 3, character: 0}
+      active: {line: 3, character: 0}
+  marks: {}
+finalState:
+  documentContents: |-
+    <?php
+
+    $myFunc = ;
```

Comment:
In typescript we would include the name as well. This feels like just one way to declare a function

Fwiw this is prob the behaviour I'd expect from "clear lambda"

Reply:
So, moving this to be "clear lambda" and removing the entire declaration would make this more consistent with other languages? I definitely would like to be consistent as much as possible.

Reply:
"Clear lambda" should do what this test case does exactly. "Clear funk" should remove the whole declaration. Make sense?

We sort of view this kind of declaration as just another way to create a named function, with the name on the LHS being its name

Reply:
Okay I moved the current functionality to the anonymous function option so this is now "change lambda"
I also added it so that "change funk" will remove the variable name as well.